repository/model/project_member.go

```diff
@@ -0,0 +1,23 @@
+package model
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+type CreateProjectMemberInput struct {
```

Comment:
@bolshakoff I think this naming feels appropriate for the repository layer after all. I'll update it in the other models as well.